back/src/modules/products/dto/products.input.ts

```diff
@@ -45,16 +45,6 @@ export class CreateProductInput {
   @Field(() => [ObjectIdScalar], { nullable: true })
   categories?: ObjectIdScalar[];
 
-  @IsOptional()
-  @IsNumber()
-  @Field(() => Float)
-  price?: number;
-
-  @IsOptional()
-  @IsNumber()
-  @Field(() => Float)
-  percentDiscount?: number;
-
   @IsOptional()
   @IsObject()
   @Field(() => ProductCharacteristicsInput, { nullable: true })
```

Comment:
**Suggestion:** It seems like you removed the price and percentDiscount fields from the CreateProductInput and UpdateProductInput classes. If these fields are no longer needed, this is fine. However, if you still need these fields, you should add them back.
```suggestion
  @IsOptional()
  @IsNumber()
  @Field(() => Float)
  price?: number;

  @IsOptional()
  @IsNumber()
  @Field(() => Float)
  percentDiscount?: number;

  @IsOptional()
  @IsObject()
  @Field(() => ProductCharacteristicsInput, { nullable: true })
```